cmd/clusterctl/client/repository/repository_local_test.go

```diff
@@ -352,28 +333,17 @@ func Test_localRepository_GetVersions(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			r, err := newLocalRepository(tt.fields.provider, tt.fields.configVariablesClient)
-			if err != nil {
-				t.Fatalf("unexpected error: %v", err)
-				return
-			}
+			g.Expect(err).NotTo(HaveOccurred())
+
 			got, err := r.GetVersions()
-			if (err != nil) != tt.wantErr {
-				t.Fatalf("error = %v, wantErr %v", err, tt.wantErr)
-			}
 			if tt.wantErr {
+				g.Expect(err).To(HaveOccurred())
 				return
 			}
-			if len(got) != len(tt.want.versions) {
-				t.Fatalf("got %v, expected %v versions", len(got), len(tt.want.versions))
-			}
-			sort.Strings(tt.want.versions)
-			sort.Strings(got)
-			for i := range got {
-				if got[i] != tt.want.versions[i] {
-					t.Errorf("got %s expected %s", got, tt.want.versions)
-				}
 
-			}
+			g.Expect(err).NotTo(HaveOccurred())
```

Comment:
nit we should be consistent in having assertion on the operation err/on wantErr like in Test_componentsClient_Get